vendor/esper-plugin-lang-java-modern.js

```diff
@@ -0,0 +1,728 @@
+/*!
```

Comment:
Should this file be in `vendor` or should it be pulled in as an external dependency?

Reply:
It could be but then we would have to do all that duck business again.